dev-lua/sol2/sol2-3.2.2.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1,3,4} luajit )
+inherit cmake lua
+
+DESCRIPTION="Header-only C++ <-> Lua API wrapper"
+HOMEPAGE="https://github.com/ThePhD/sol2"
+SRC_URI="https://github.com/ThePhD/sol2/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+REQUIRED_USE="${LUA_REQUIRED_USE}"
+RESTRICT="!test? ( test )"
+
+RDEPEND="${LUA_DEPS}"
+DEPEND="
+	test? (
+		dev-cpp/catch:1
```

Comment:
Should this really be catch:1?

regardless of gcc11 or 10, I get this:
```
/tmp/portage/dev-lua/sol2-3.2.2/work/sol2-3.2.2/tests/runtime_tests/source/utility.cpp: At global scope:
/tmp/portage/dev-lua/sol2-3.2.2/work/sol2-3.2.2/tests/runtime_tests/source/utility.cpp:126:24: error: expected constructor, destructor, or type conversion before '(' token
  126 | TEMPLATE_LIST_TEST_CASE("utility/optional-conversion", "test that regular optional will properly catch certain types", OptionalsToTest) {
      |                        ^
```

But solved if I replace this by catch:0 (aka catch2), and adjust the include dir to catch2.

Reply:
It works for me with both. But they used version 1, so I did the same. Updated.